nginx/certs/fullchain.pem

```diff
@@ -0,0 +1,24 @@
+-----BEGIN CERTIFICATE-----
+MIIECzCCAvOgAwIBAgIJAOnZtZjMBrtKMA0GCSqGSIb3DQEBCwUAMIGbMQswCQYD
+VQQGEwJVUzETMBEGA1UECAwKQ2FsaWZvcm5pYTEUMBIGA1UEBwwLTG9zIEFuZ2Vs
+ZXMxFTATBgNVBAoMDFR5cmVsbCBDb3JwLjEQMA4GA1UECwwHTmV4dXMtNjEUMBIG
+A1UEAwwLZXhhbXBsZS5vcmcxIjAgBgkqhkiG9w0BCQEWE3IuYmF0dHlAZXhhbXBs
+ZS5vcmcwHhcNMjAwNTMxMTkxNDUzWhcNMjEwNTMxMTkxNDUzWjCBmzELMAkGA1UE
+BhMCVVMxEzARBgNVBAgMCkNhbGlmb3JuaWExFDASBgNVBAcMC0xvcyBBbmdlbGVz
+MRUwEwYDVQQKDAxUeXJlbGwgQ29ycC4xEDAOBgNVBAsMB05leHVzLTYxFDASBgNV
+BAMMC2V4YW1wbGUub3JnMSIwIAYJKoZIhvcNAQkBFhNyLmJhdHR5QGV4YW1wbGUu
+b3JnMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAwjbOYsTdeelOlAqq
+Sh6cSDZTGmFoUSwLeNs2zmgLuECpJYZYJLYBMWGu6SBjI9lc85uWU+Zgj9uiWfVr
+8NgOZ+8R+8K40Yn76x5m3ypnnAbLQjzE+F1/sL9l80A0avKGPXtoL+h4VrAPWo37
+QONuPFc+dvfhnsKeMnu6ikUu4ztNoajdQ9/R6jUV2sAUPlQJw/rjt+eOc22K05yZ
+mvxCZOMRD22nCnUlyrh0xJRdC5XS5sViURqjBTMf9nJ5shrW6IOAHk3SaW7TGlvU
+zd7GNIkL+UK/vsLjiwIty3QqLDVDDJQ37kh6LtttGuXyyLNsdEKQGSBJpZmKL0fn
+YevXpQIDAQABo1AwTjAdBgNVHQ4EFgQUrhIFrYs6i0cv9zsUXfFvd9mP8rswHwYD
+VR0jBBgwFoAUrhIFrYs6i0cv9zsUXfFvd9mP8rswDAYDVR0TBAUwAwEB/zANBgkq
+hkiG9w0BAQsFAAOCAQEAFbocf0WG+6SCOeOoIHBqLBqPdLxYYXTNVIFhDJfzpjV5
+mVHHK+VTBQ5FBbgtIzoQrl7COTTBM1aPN0antMzdJ+dtdFPaG0BEih2tcM6QdXAy
+j0cCLvdXQBmK/Qvq8Dn4hYTQTggXAncDuXImYmPPbwp5oq22oEThT9KUzD2BwnFp
+g+FlQa/MMP5xpnCgzpiylPa3Tpj0AqAC5o7IMJgFo5xKVfZFlRe73wPWkzU/wUNw
+MAEDpbu0vmM682encg2M/cLYSClmppUYOu8wTQ5lOeGzwmJVbuM/78C4RZLZlncI
+0Yx3Us4+3lZcQJy4SIOnxMTBPBX8udyvxFWUkd8M8w==
+-----END CERTIFICATE-----
```

Comment:
We should have some docs how we generated these :) Or a script for re-producibility.

Reply:
Added to the todo list :rocket: